data/power.quirk

```diff
@@ -8,3 +8,7 @@ BatteryThreshold = 30
 
 [HP]
 BatteryThreshold = 50
+
+# Framework
+[e5319542-ca16-5d93-bd0f-2d25f547a846]
```

Comment:
where did this come from?  The HWID I had was:
```
{2185e781-ed4b-5fc2-8839-b5ddb1e7d649}   <- Manufacturer
```

Reply:
Ohh wowsers, not sure. Good catch!